packages/shared/src/components/cards/v1/TypeLabel.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { PostType } from '../../../graphql/posts';
+
+type TypeLabelType = PostType | ReactElement | string;
+
+const typeToClassName: Record<PostType, string> = {
```

Comment:
Why do we have the ones that don't have one defined still in here?

Reply:
Originally I thought that we will have a style for every type and later forgot to remove the unused ones.